src/components/forms/transaction/SplitField.tsx

```diff
@@ -144,7 +145,7 @@ export default function SplitField({
             step="0.001"
             disabled={disabled}
             onChange={(e) => {
-              if (account.commodity.guid === txCurrency.guid) {
+              if (!showValueField && splits.length > 1) {
```

Comment:
When it is an update, we display the single split of a split event as a SplitField and not a MainSplit. This is to avoid the quantity being updated